cogs/verification.py

```diff
@@ -0,0 +1,138 @@
+import aiohttp
+import json
+from random import choice, choices, randint, sample
+
+from discord import File, Error
+from discord.ext.commands import (
+    Cog,
+    Bot,
+    Context,
+    command,
+    CommandInvokeError,
+    has_permissions,
+)
```

Comment:
Consider using a better import structure:
```suggestion
from discord.ext.commands import (Bot, Cog, CommandInvokeError, Context,
                                  command, has_permissions)
```